### PR TITLE
Add functionality for select/deselect all in MultiSelectDropDown

### PR DESCRIPTION
**Pull request by theref**

### CODE REVIEW

**Comment by meatballs** [comment]
Nice!

Could we have an entry in the [changelog](https://github.com/anvilistas/anvil-extras/blob/main/CHANGELOG.md) and something in the [docs](https://github.com/anvilistas/anvil-extras/blob/main/docs/guides/components/multi_select_dropdown.rst) for this?

**Review by s-cork — APPROVED** [review]
Code looks good to me 😊

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # Unreleased
+* Select All functionality added to MultiSelect component
```
Comment (meatballs) [inline CHANGELOG.md:2]: Final nitpick - Can you move this into the New Features section and then I reckon this is ready to merge.

**Comment by meatballs** [comment]
Excellent! Thanks for this @theref

**Pull request merged**